lib/proto/attach-form.js

```diff
@@ -41,7 +41,7 @@ var formAction = function(el, event) {
     // jscs:disable disallowImplicitTypeConversion
     if (!!element.name && element.attributes !== undefined && tagName !== "button") {
       var type = element.attributes.type
-      // jscs:enable disallowImplicitTypeConversion
```

Comment:
Did you meant to remove this?

Reply:
Yes. The tests run fine without it, so it's not necessary.

I should have added it to #147, but I forgot to do it before I merged it.